language-guide/generics.md

```diff
@@ -312,7 +312,7 @@ func findIndex<T: Equatable>(of valueToFind: T, in array:[T]) -> Int? {
 
 ```swift
 let doubleIndex = findIndex(of: 9.3, in: [3.14159, 0.1, 0.25])
-// doubleIndex は、9.3 が配列にないため、オプショナルの Int です。
+// doubleIndex は、9.3 が配列にないため、オプショナルの Int 型で値はありません。
```

Comment:
原文は以下です。
https://docs.swift.org/swift-book/LanguageGuide/Generics.html
```swift
let doubleIndex = findIndex(of: 9.3, in: [3.14159, 0.1, 0.25])
// doubleIndex is an optional Int with no value, because 9.3 isn't in the array
```

`with no value`の情報が現状抜け落ちていたので補いました